java-lib/src/main/java/com/wavefront/ingester/AbstractIngesterFormatter.java

```diff
@@ -44,6 +51,7 @@ protected AbstractIngesterFormatter(List<FormatterElement<T>> elements) {
 
   protected interface FormatterElement<T> {
     void consume(StringParser parser, T target);
```

Comment:
do 

```
default void consume(StringParser parser, T target) {
  consume(parser, target, DEFAULT_INGESTER_CONTEXT);
}
```

In general, avoid interface changes that are breaking (now that we have default)